### PR TITLE
Style improvements

### PR DESCRIPTION
**Pull request by greg0ire**
Cleans up the mess I introduced in #2835 and #2851 

@SenseException @lcobucci please review preferrably before this gets merged :P

### CODE REVIEW

File: tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php [hunk 1]
```diff
@@ -563,16 +563,16 @@ public function testGetDefaultValueDeclarationSQLForIntegerTypes()
...
-        $field = array(
+        $field = [
             'type'    => Type::getType('date'),
             'default' => $currentDateSql,
-        );
+        ];
 
-        $this->assertEquals(
-            ' DEFAULT '.$currentDateSql,
+        $this->assertSame(
```
Comment (Majkl578) [inline tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php]: Should use `self::`.

File: tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php [hunk 2]
```diff
@@ -563,16 +563,16 @@ public function testGetDefaultValueDeclarationSQLForIntegerTypes()
         }
     }
 
-    public function testGetDefaultValueDeclarationSQLForDateType()
+    public function testGetDefaultValueDeclarationSQLForDateType(): void
```
Comment (Majkl578) [inline tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php]: Space around `:`.

**Review by Majkl578 — CHANGES_REQUESTED** [review]

**Comment by SenseException** [comment]
Does Doctrine have a coding standard about concatenation of strings with `.`? Spaces or no spaces?

**Comment by Ocramius** [comment]
Spaces

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by SenseException** [comment]
I have nothing to add here. :+1:

**Comment by lcobucci** [comment]
@SenseException you can see more info on https://github.com/doctrine/coding-standard

**Pull request merged**